src/hugr/views/hierarchy.rs

```diff
@@ -345,7 +344,7 @@ where
     }
 }
 
-/// A common trait for views of a hugr region.
+/// A common trait for views of a HUGR hierarchical subgraph.
```

Comment:
nit: "for views of subgraphs of a Hugr that" and then something like "include all descendants/subtrees" or "are closed over children"